builtin/credential/jwt/path_config.go

```diff
@@ -586,6 +659,52 @@ func (c jwtConfig) hasType(t string) bool {
 	return strutil.StrListContains(c.OIDCResponseTypes, t)
 }
 
+func fipsAllowedChains(chains [][]*x509.Certificate) ([][]*x509.Certificate, error) {
+	if !fips140.Enabled() {
```

Comment:
This function was replicated exactly, with the exception of this line. This line was modified to use a publicly exposed method.

Reply:
If you don't mind adding a comment about which version of the Go source you pulled this from, that'd be appreciated :-) 

Something like: https://github.com/openbao/openbao/blob/main/builtin/logical/pki/path_root.go#L630